rotkehlchen/types.py

```diff
@@ -837,6 +838,57 @@ def __str__(self) -> str:
         return f'Addressbook entry with name "{self.name}", address "{self.address}" and blockchain {str(self.blockchain) if self.blockchain is not None else None}'  # noqa: E501
 
 
+class LocationAssetMappingDeleteEntry(NamedTuple):
+    location: 'Location | None'
+    location_symbol: str
+
+    def serialize(self) -> dict[str, str | None]:
+        return {
+            'locatoin': None if self.location is None else self.location.serialize(),
```

Comment:
Hm this typo was not seen in the tests? Or are tests also wrong?

```suggestion
            'location': None if self.location is None else self.location.serialize(),
```